### DISCUSSION TITLE
Airbyte not showing up on DagIT

### DISCUSSION THREAD

**Discussion by Resilienciaftw** [dagster-io/dagster#19904] [Q&A] [answered]
I have an Airbyte Instance running. I created a new dagster project within my dbt project with command 
`dagster-dbt project scaffold --name orchestration`
When I run dagster dev, the dbt models show up in DagIt. 

For Airbyte connection, I added new file airbyte.py and added following code:
`from dagster_airbyte import airbyte_resource,load_assets_from_airbyte_instance

from .constants import AIRBYTE_CONNECTION_ID, AIRBYTE_CONFIG

airbyte_instance = airbyte_resource.configured(AIRBYTE_CONFIG)

airbyte_assets = load_assets_from_airbyte_instance(
    airbyte_instance,
    workspace_id=AIRBYTE_CONNECTION_ID,
    )`

I have connection_ID [ as I have multiple connections in Airbyte, I used the connection ID parameter ] and airbyte configs [ localhost, 8000, username and password ] defined in file constants.py

When I run dagster dev, DagIt runs without error and shows DBT models but does not show Airbyte assets. Am I missing something here?

![image](https://github.com/dagster-io/dagster/assets/155509684/5e739db4-3be7-427a-b894-e3cca8fe18b9)

**Comment by rexledesma** [comment, marked as answer]
You need to add `airbyte_assets` to the `assets` argument of your [`Definitions`](https://docs.dagster.io/_apidocs/definitions) object. See https://docs.dagster.io/integrations/airbyte#step-3-adding-downstream-assets as an example.
